symengine/lib/symengine_wrapper.pyx

```diff
@@ -4369,7 +4369,7 @@ cdef class _Lambdify(object):
     cdef vector[int] accum_out_sizes
     cdef object numpy_dtype
 
-    def __init__(self, args, *exprs, cppbool real=True, order='C', cppbool cse=False):
+    def __init__(self, args, *exprs, cppbool real=True, order='C', cppbool cse=False, cppbool load=False):
```

Comment:
maybe we should name the kwarg `_load` to signal that end-users should be touching it?

Reply:
Good suggestion. I'll change this